### DISCUSSION TITLE
MacOS release

### DISCUSSION THREAD

**Discussion by sergeysova** [mulaRahul/keyviz#12] [Ideas]
The design looks beautiful. What about to release for macOS users?

**Comment by mulaRahul** [comment]
Of course, that's definitely on the roadmap!
Unfortunately, I'm a Windows user and don't have access to a Mac system, so have to wait until I get my hands on one 😅.

**Reply by ImBIOS** [reply]
I can help you with it

**Comment by mulaRahul** [comment]
@sergeysova, you can checkout the alpha release [here](https://github.com/mulaRahul/keyviz/releases/tag/v2.0.0a).
